bin/bit.js

```diff
@@ -32,6 +34,16 @@ function verifyCompatibility() {
   return true;
 }
 
+function bitVersion() {
+  if (process.argv[2]) {
+    if (process.argv[2] === '-V' || process.argv[2] === '-v' || process.argv[2] === '--version') {
+      const BIT_VERSION = require('../dist/constants').BIT_VERSION;
```

Comment:
The constants are already required few lines below, just use the same require and move it p

Reply:
done :)